src/test/cli.build.test.ts

```diff
@@ -170,5 +170,61 @@ describe('Dev Containers CLI', function () {
 			assert.equal(response.imageName[0], image1);
 			assert.equal(response.imageName[1], image2);
 		});
+
+		it('should fail with only --output-type and no --output-dest', async () => {
```

Comment:
Would it make sense to have a test that checks a tar is created at the output location when the cli options are correct?

Reply:
Good call. On it.